code/game/objects/structures/teleporter.dm

```diff
@@ -141,16 +153,28 @@
 	///The optional cell to power the teleporter if off the grid
 	var/obj/item/cell/cell
 	COOLDOWN_DECLARE(teleport_cooldown)
+	
+	///tag to avoid mixups. Purely for players, not for systems.
+	var/static/tele_tag = 78
+	///Easier reference to the correct number.
```

Comment:
autodoc should be self contained, not reference to another autodoc

Reply:
Done, I think. If this doesn't work, I wouldn't mind getting an example of what you'd want.